pkg/controller/component/component_controller.go

```diff
@@ -77,9 +98,6 @@ type ReconcileComponent struct {
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileComponent) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
-	reqLogger.Info("Reconciling Component")
-
 	// Fetch the Component instance
 	instance := &devconsoleapi.Component{}
```

Comment:
How about using `cr` as the variable name for Component instead of too generic `instance`?

Reply:
yes we could, let's do it in separate PR thou to avoid too many conflicts given we have some pending PR in the same code area.